src/views/shared/SecurityBeacon.js

```diff
@@ -64,7 +67,14 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         viewport: $('body')
       },
       hide: {event: false, fixed: true},
-      show: {event: false, delay: 200}
+      show: {event: false, delay: 200},
+      events: {
+        move: function(event, api) {
```

Comment:
I see... What if the window is resized again and the security image is visible again? Maybe we should hide/show the qtip based on the same css as we hide/show the security image when we resize the window. Can you look into that please?